IRC.md

```diff
@@ -1,111 +1,238 @@
 # Connecting to IRC
 
-Twitch offers an IRC interface to its chat. This allows for people to do things like develop bots for their channel, or to connect to a channel's chat with an IRC client instead of using the web interface. While our IRC server is built to follow [RFC1459](http://tools.ietf.org/html/rfc1459.html), it is important to note that there are some cases where it will behave slightly differently than another IRC server would. These cases are noted when necessary in the following document.
+Twitch offers an IRC interface to its chat. This allows for people to do things like develop bots for their channel, or to connect to a channel's chat with an IRC client instead of using the web interface. While our IRC server is built to follow [RFC1459](http://tools.ietf.org/html/rfc1459.html), it is important to note that there are **several** cases where it will behave slightly differently than another IRC server would. These cases are noted when necessary in the following document.
 
 Lines prefixed with < are sent from client to server, and lines prefixed with > are sent from the server to the connecting client.
 
 ## Prerequisites
+
 In order to connect to Twitch IRC, you must have three pieces of information:
 
 1. The name of channel that you want to join.
 2. A Twitch account.
-3. Oauth token from API or from a site using it: www.twitchapps.com/tmi 
+3. An OAuth token [authorized through our API](/authentication.md) either directly or via an application which provides it
 
 ## Connecting
+
 Once you have that information, you can then take it to connect to Twitch IRC with the following bits of information:
 
 - The server name to connect to is: *irc.twitch.tv*.
-- The port to connect to is *6667*.
-- SSL **is not** supported for Twitch IRC.
+- SSL is not currently supported for Twitch IRC
```

Comment:
yeah this was because I had linked to a comment that we will support ssl in the future, but then removed the comment and forgot to change the ordering back